apps/fluent-tester/src/TestComponents/Shadow/ShadowButtonTestSection.tsx

```diff
@@ -16,9 +23,15 @@ export const ShadowButtonTestSection: React.FunctionComponent = () => {
   if (Platform.OS === 'ios') {
     return (
       <View style={shadowTestPageStyles(t).backgroundColor}>
-        <FAB style={commonTestStyles.vmargin}>FAB with default shadow</FAB>
-        <CustomFABShadow64 style={commonTestStyles.vmargin}>Custom FAB with shadow64</CustomFABShadow64>
-        <CustomFABNoShadow style={commonTestStyles.vmargin}>Custom FAB with no shadow</CustomFABNoShadow>
+        <FAB icon={{ svgSource: svgProps }} style={commonTestStyles.vmargin}>
```

Comment:
>{ svgSource: svgProps } [](http://example.com/codeflow?start=19&length=23)

Please make this into a const object outside of the function component so it doesn't generate a new object on each render

Reply:
Also why is this change necessary?

Reply:
FAB must have an icon, have updated the type to make it a required prop and hence had to add it here as well.